ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -610,7 +654,12 @@ export const PageProviderDisplayBehavior = [
               this.set(`items.${i}`, response.entries[entryIndex++]);
 
               if (isSelected) {
-                this.selectIndex(i);
+                if (this.selectAllChecked) {
+                  this.set(`selectedItems.${i}`, this.items[i]);
+                  this._selectItemModel(i)
```

Comment:
missing a semicolon there
```suggestion
                  this._selectItemModel(i);
```

Reply:
Thank you 👍